examples/simple/components/Icon.js

```diff
@@ -51,6 +51,7 @@ export default class Icon extends Component {
     color: React.PropTypes.string,
     height: React.PropTypes.number,
     kind: React.PropTypes.oneOf([
+    className: PropTypes.string,
```

Comment:
`className` is rendered inside the array for `kind` propType...

can you also modify `react-svg-icon-generator/examples/simple/template/icon.nunjucks` template file?